### PR TITLE
Job methods

### PR DESCRIPTION
**Pull request by ZeeshanJan**
Added / Updated the functions required.

### CODE REVIEW

File: contracts/DataModel.sol [hunk 1]
```diff
@@ -30,6 +30,8 @@ library FrontDoorStructs {
     struct Job {
         uint256 id;
         uint256 bounty;
+        bool status;
```
Comment (ppelicano) [inline contracts/DataModel.sol:33]: Just a suggestion, I think status is a bit too generalistic meaning. I believe something live isRemoved would read better. But your call it is fine like this as well :)

File: contracts/DataModel.sol [hunk 2]
```diff
@@ -40,6 +42,11 @@ library FrontDoorStructs {
         Job job;
     }
 
+    struct Company{
+        address wallet;
+        uint256 jobsCreated;
```
Comment (ppelicano) [inline contracts/DataModel.sol:47]: Why do we need an incrementor variable. Is it for future use? or does it have a function already now ??? 👍

**Review by ppelicano — APPROVED** [review]

File: contracts/DataModel.sol (hunk 1 above)
Comment (ZeeshanJan) [reply contracts/DataModel.sol:33]: Sure, I have updated it - will make a commit & push now.

File: contracts/DataModel.sol (hunk 2 above)
Comment (ZeeshanJan) [reply contracts/DataModel.sol:47]: It works as a counter; that a company has created X number of jobs on our platform. We may also keep track of a company-specific jobs created (for example, an array of job-ids created by a company).

We can remove it if you think it isn't needed.

**Pull request merged**